<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="moveForwardButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRotateLeftButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRotateRightButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveBackwardsButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Backwards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IPaddressTextbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::WidgetWidth</enum>
    </property>
    <property name="maximumBlockCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="portTextbox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="maximumBlockCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <zorder>moveForwardButton</zorder>
   <zorder>moveRotateLeftButton</zorder>
   <zorder>moveRotateRightButton</zorder>
   <zorder>moveBackwardsButton</zorder>
   <zorder>IPaddressTextbox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>portTextbox</zorder>
   <zorder>connectButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
